src/util/ratingTypes.ts

```diff
@@ -18,3 +18,29 @@ export interface Rating {
 	dateCreated: Date;
 	dateEdited?: Date;
 }
+
+export enum RatingValue {
+	ONE = 1,
+	ONE_HALF = 1.5,
+	TWO = 2,
+	TWO_HALF = 2.5,
+	NOT_ASSESSED = 3,
+	THREE = 3.5,
+	THREE_HALF = 4,
+	FOUR = 4.5,
+	FOUR_HALF = 5,
+	FIVE = 5.5,
+}
```

Comment:
I'm confused - why would we use 3 as the default and skip over it as a number?

The default should just be `null`